### ISSUE TITLE
Can we train cycle GAN using paired dataset (Aligned dataset)

### ISSUE THREAD

**Issue by shivom9713** [junyanz/pytorch-CycleGAN-and-pix2pix#1230]
Sir , if we have a aligned dataset , can cycleGAN also show supervised learning behaviour ?
I mean can we add L1 loss between ground truth image (domain B) and generated image (domain B).
That is in simple words is it possible to include pix2pix kind of L1 loss using aligned dataset into CycleGAN for better results

**Comment by JunlinHan** [comment]
Definitely, you can. But I assume you will get inferior results compared to Pix2Pix.
Aligned dataset is easier for the training of network. Using aligned dataset for training may reduce mode collapse.

**Comment by shivom9713** [comment]
You mean cycle GAN when trained with aligned dataset shows supervised behaviour  and can we add more supervised losses in it??
